File: backend/models/hospital.py
from sqlalchemy import (
    Column,
    String,
    Integer,
)
from db.configurations import Base

class Hospital(Base):
    __tablename__ = "hospital"
    hospitalID = Column(Integer,primary_key=True,autoincrement=True)
    hospitalName = Column(String(50),nullable=False)
    hospitalContact = Column(Integer,nullable=False)
    hospitalAddress = Column(String(50),nullable=False)

File: frontend/src/types/hospital.ts
export type HospitalDropdown = {
    hospitalID: string,
    hospitalName: string
}

export type HospitalDetails = HospitalDropdown & {
    hospitalContact: number,
    hospitalAddress: string
}

export type HospitalAssociation = {
    hospitalID: number,
    doctorID: number
}